packages/vike-react/renderer/types.ts

```diff
@@ -1,9 +1,26 @@
-export type { Component }
+export type { Component, PageView }
 
 import type { ReactElement } from 'react'
+import type { Pipe } from 'react-streaming/dist/cjs/server/renderToStream/createPipeWrapper'
 
 // type Component = (props: Record<string, unknown>) => ReactElement
 type Component = (props: any) => ReactElement
+type PageView = string | { _escaped: string } | StreamResult
```

Comment:
Is this correct? is it ok we use type like this? or we don't need it?

Reply:
You could also do something like this instead

```
type PageView = string | Awaited<ReturnType<typeof renderToStream>>

Reply:
Sure